apps/mobile/src/features/send/send-form/components/send-form-numpad.tsx

```diff
@@ -0,0 +1,17 @@
+import { useFormContext } from 'react-hook-form';
+
+import { Box, Numpad } from '@leather.io/ui/native';
+
+import { SendFormSchema } from '../send-form.schema';
+
+export function SendFormNumpad() {
+  const { setValue, watch } = useFormContext<SendFormSchema>();
+
+  const amount = watch('amount');
```

Comment:
not super clear what `watch` does

Reply:
Docs for `watch`: https://react-hook-form.com/docs/useform/watch